src/components/avatars/Avatar.stories.js

```diff
@@ -0,0 +1,58 @@
+// import MyButton from './Button.vue';
+import ImgAvatar from "./index.vue"
+import { action } from "@storybook/addon-actions"
+
+// More on default export: https://storybook.js.org/docs/vue/writing-stories/introduction#default-export
+export default {
+  title: 'Example/ImgAvatar',
+  component: ImgAvatar,
```

Comment:
image component name should start with prefix of dd-

Reply:
I resolved this issue now.